pom.xml

```diff
@@ -59,8 +59,9 @@
         </repository>
     </distributionManagement>
     <properties>
-        <java.version>1.6</java.version>
-        <java.testversion>1.8</java.testversion>
+        <maven.compiler.release>11</maven.compiler.release>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
```

Comment:
If the `maven.compiler.release` property is used the older `maven.compiler.source` and `maven.compiler.target` properties should be omitted.